doc/masking.rst

```diff
@@ -0,0 +1,97 @@
+
+The following procedure is used, note these steps must be executed one after
+another, without LV power cycle or action to cause a reset of the VFAT settings
+(e.g. SCA reset):
+
+======= ===================================================== ========= ==================================================== ======
+Step    Tool ``v2b`` (``v3``)                                 VFAT Data Input Config                                         Generates
+======= ===================================================== ========= ==================================================== ======
+1       ``trimChamber.py`` (``trimChamberV3.py``)             Tracking  ``VThreshold1 (CFG_THR_ARM_DAC) = 100, ztrim=4``     Initial channel configuration ``chConfig.txt`` and ``trimRange`` settings.
+2       ``confChamber.py``                                    N/A       ``chConfig.txt``, ``trimRange in memory``            Nothing
+3       ``ultraThreshold.py``                                 Tracking  Nothing                                              Updated channel config ``chConfig_MasksUpdated.txt`` and initial VFAT settings storing VThreshold1 and trimRange in vfatConfig.txt.
+4       ``confChamber.py``                                    N/A       ``chConfig_MasksUpdated.txt`` and ``vfatConfig.txt`` Nothing
+5       ``ultraThreshold.py`` (``sbitThreshScanParallel.py``) Trigger   Nothing                                              Updated VFAT settings ``vfatConfig_Updated.txt`` with final ``VThreshold1`` values.
+======= ===================================================== ========= ==================================================== ======
+
+Please note that while ``DeadChannel`` is given in ``maskReason``, these
+channels are never masked such that they can be tracked over time.
+
+If a channel was masked at the time of acquisition of a test involving an
+S-curve measurement (e.g. ``trimChamber(V3).py`` or ``ultraScurve.py``), it will
+be assigned the ``FitFailed`` reason since the original reason is not known
+without referencing a previous scan.
+
+Providing cuts for ``maskReason`` at runtime
+--------------------------------------------
+
+When analyzing the above S-curves taken by ``trimChamber(V3).py``, the following
+command line arguments are available for specifying the cut values for assigning
+the ``DeadChannel``, ``HighNoise``, and ``HighEffPed`` pedestal:
+
+.. option:: --maxEffPedPercent <NUMBER>
+
+    Value from 0 to 1. Threshold for setting the ``HighEffPed`` mask reason, if
+    channel ``effPed > maxEffPedPercent * nevts`` then ``HighEffPed`` is set.
+
+.. option:: --highNoiseCut <NUMBER>
+
+    Threshold for setting the ``HighNoise`` ``maskReason``, if channel
+    ``scurve_sigma > highNoiseCut`` then ``HighNoise`` is set.
+
+.. option:: --deadChanCutLow <NUMBER>
+
+    If channel ``deadChanCutLow < scurve_sigma < deadChanCutHigh`` then
+    ``DeadChannel`` is set, see slide 22 of `this talk`_ for the origin of the
```

Comment:
Might want to mention that `this_talk` is only for `vfat2` and a separate set of values ("coming soon") are needed for `vfat3`.